backend/src/test/java/ai/verta/modeldb/ProjectTest.java

```diff
@@ -3625,4 +3630,61 @@ public void deleteProjectArtifacts() {
 
     LOGGER.info("Delete Project Artifacts test stop................................");
   }
+
+  @Test
+  public void createProjectWithGlobalSharingOrganization() {
+    LOGGER.info("Global organization Project test start................................");
+
+    if (app.getAuthServerHost() == null || app.getAuthServerPort() == null) {
+      Assert.assertTrue(true);
+      return;
+    }
+
+    ProjectServiceBlockingStub projectServiceStub = ProjectServiceGrpc.newBlockingStub(channel);
+    OrganizationServiceGrpc.OrganizationServiceBlockingStub organizationServiceBlockingStub =
+        OrganizationServiceGrpc.newBlockingStub(authServiceChannelClient1);
+
+    String orgName = "Org-test-verta";
+    SetOrganization setOrganization =
+        SetOrganization.newBuilder()
+            .setOrganization(
+                Organization.newBuilder()
+                    .setName(orgName)
+                    .setDescription("This is the verta test organization")
+                    .build())
+            .build();
+    SetOrganization.Response orgResponse =
+        organizationServiceBlockingStub.setOrganization(setOrganization);
+    Organization organization = orgResponse.getOrganization();
+    assertEquals(
+        "Organization name not matched with expected organization name",
+        orgName,
+        organization.getName());
+
+    // Create project
+    CreateProject createProjectRequest = getCreateProjectRequest("project_n_sprt");
+    createProjectRequest =
+        createProjectRequest
+            .toBuilder()
+            .setWorkspaceName(organization.getName())
+            .setProjectVisibility(ProjectVisibility.ORG_SCOPED_PUBLIC)
+            .build();
+    CreateProject.Response createProjectResponse =
+        projectServiceStub.createProject(createProjectRequest);
+    Project project = createProjectResponse.getProject();
+    LOGGER.info("Project created successfully");
```

Comment:
can we check if the appropriate role is created as well? same for other test

Reply:
Ok. I can add it.

Reply:
Added Role check after creating the organization in test cases